agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/AiConfigCustomizer.java

```diff
@@ -182,10 +182,13 @@ private static void enableInstrumentations(
         "otel.instrumentation.opentelemetry-instrumentation-annotations.enabled", "true");
     properties.put("otel.instrumentation.opentelemetry-api.enabled", "true");
     properties.put("otel.instrumentation.opentelemetry-instrumentation-api.enabled", "true");
-    if (otelConfig.getBoolean("otel.instrumentation.reactor.enabled", true)) {
+    if (otelConfig.getBoolean("otel.instrumentation.reactor.enabled", true)
+        && !"false".equalsIgnoreCase(System.getenv("OTEL_INSTRUMENTATION_REACTOR_ENABLED"))) {
```

Comment:
what is the equivalent of system property for OTEL_INSTRUMENTATION_REACTOR_ENABLED?